src/main/java/software/amazon/smithy/lsp/ext/Completions.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit: Wrong copyright year on the new files.

Reply:
To be clear, do you want all copyright notices to be updated to 2022, or you'd like files that we introduced to have our (disney streaming) Copyright notice?